src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1187,6 +1187,47 @@ public static void mutateMV(ByteBuffer dataKey, Collection<Mutation> mutations,
         }
     }
 
+    private static ConsistencyLevel consistencyLevelForCommit(Collection<? extends IMutation> mutations, ConsistencyLevel consistencyLevel)
+    {
+        ConsistencyLevel result = null;
+        for (IMutation mutation : mutations)
```

Comment:
We'll have to give this some thought, since that will mean splitting up mutations that write to multiple tables, as well as making write coordination concurrent so that we coordinate normal and accord writes simultaneously

Reply:
I think that might be enough of an undertaking to warrant punting it to a follow on ticket. I also wonder if we could get away with just having this restriction for interop.... basically forbidding multi-table / multi-key batches when using accord, or migrating to/from it. It is something we discourage

Reply:
I recall from discussions with Cloudkit and reviewing their queries that they definitely use unlogged batches and the use case is to split data over multiple partitions so they don't run into the large partition problem.

Reply:
I confirmed that Cloudkit definitely does batches to multiple tables and partitions from the full query log I took a while back.

Reply:
deferred to https://issues.apache.org/jira/browse/CASSANDRA-19431